.github/workflows/tests.yml

```diff
@@ -36,3 +36,27 @@ jobs:
         python3 -m pytest .
       env:
         CALCAT_OAUTH_CREDS: ${{ secrets.CALCAT_OAUTH_CREDS }}
+
+  publish:
```

Comment:
shouldn't this block be indented under `jobs`?

Reply:
It is, but just with two spaces. :slightly_smiling_face: